test_pawn.py

```diff
@@ -0,0 +1,33 @@
+from game import Game
+
+"""Pawn Moves Test"""
+
+
+def test_indices_to_coordinates():
+    state = Game()
+    i, j = 6, 3
+    result = state.board.idxs_to_coordinates(i, j)
+    assert result == "E7"
+
+
+def test_forward():
+    state = Game()
+    for col in state.board.columns.keys():
```

Comment:
you should add a comment on what this is testing. One of the things about testing, is that the tests should show 1) be apparent to what you are testing 2) show how you use the functions. I don't really understand why we have a for loop (or it isn't intuitive) whereas the `test_forward_and_diagnoal` does not have a for loop. So either 1) we don't really need a for loop or 2) you should document why we need a for loop. 

In the documentation for the for loop, please also add that the keys are iterating through A-H

Also, I believe this is testing that on the first move, the pawn can move one or two spaces forward. You should also document this in the test. You should also test that if it is not the first move, it can only move one space forward (by maybe changing `first_move` to false?)